src/compass/s1_geocode_metadata.py

```diff
@@ -122,6 +127,132 @@ def run(cfg, burst, fetch_from_scratch=False):
         f"{module_name} burst successfully ran in {dt} (hr:min:sec)")
 
 
+def geocode_calibration_luts(geo_burst_h5, burst, cfg, 
+                             dec_factor=40):
+    '''
+    Geocode the radiometric calibratio paremeters,
+    and write them into output HDF5.
+
+    Parameters
+    ----------
+    geo_burst_h5: h5py.files.File
+        HDF5 object as the output product
+    burst: s1reader.Sentinel1BurstSlc
+        Sentinel-1 burst SLC
+    cfg: GeoRunConfig
+        GeoRunConfig object with user runconfig options
+    dec_factor: int
+        Decimation factor to downsample the slant range pixels for LUT
+    '''
+    dem_raster = isce3.io.Raster(cfg.dem)
+    epsg = dem_raster.get_epsg()
+    proj = isce3.core.make_projection(epsg)
+    ellipsoid = proj.ellipsoid
+    burst_id = str(burst.burst_id)
+    geo_grid = cfg.geogrids[burst_id]
+    radar_grid = burst.as_isce3_radargrid()
+
+    date_str = burst.sensing_start.strftime("%Y%m%d")
+    burst_id_date_key = (burst_id, date_str)
+    out_paths = cfg.output_paths[burst_id_date_key]
+
+    # Common initializations
+    threshold = cfg.geo2rdr_params.threshold
+    iters = cfg.geo2rdr_params.numiter
+    scratch_path = out_paths.scratch_directory
+
+    # Designate radiometric calibration parameter to geocode
+    calibration_dict = {
+        'gamma':burst.burst_calibration.gamma,
+        'sigma_naught':burst.burst_calibration.sigma_naught,
+    }
+
+    # define the geogrid for calbration LUT
+    calibration_radargrid = radar_grid.multilook(dec_factor,
+                                                 dec_factor)
+    calibration_geogrid = isce3.product.GeoGridParameters(
+                            geo_grid.start_x,
+                            geo_grid.start_y,
+                            geo_grid.spacing_x * dec_factor,
+                            geo_grid.spacing_y * dec_factor,
+                            geo_grid.width // dec_factor + 1,
+                            geo_grid.length // dec_factor + 1,
+                            geo_grid.epsg)
+
+    # init geocode object
+    geocode_obj = isce3.geocode.GeocodeFloat32()
+    geo_calibration.orbit = burst.orbit
+    geo_calibration.ellipsoid = ellipsoid
+    geo_calibration.doppler = isce3.core.LUT2d()
+    geo_calibration.threshold_geo2rdr = threshold
+    geo_calibration.numiter_geo2rdr = iters
+    geo_calibration.geogrid(geogrid_calibration.start_x,
+                            geogrid_calibration.start_y,
+                            geogrid_calibration.spacing_x,
+                            geogrid_calibration.spacing_y,
+                            geogrid_calibration.width,
+                            geogrid_calibration.length,
+                            geogrid_calibration.epsg)
```

Comment:
```suggestion
    geocode_obj.orbit = burst.orbit
    geocode_obj.ellipsoid = ellipsoid
    geocode_obj.doppler = isce3.core.LUT2d()
    geocode_obj.threshold_geo2rdr = threshold
    geocode_obj.numiter_geo2rdr = iters
    geocode_obj.geogrid(calibration_geogrid.start_x,
                        calibration_geogrid.start_y,
                        calibration_geogrid.spacing_x,
                        calibration_geogrid.spacing_y,
                        calibration_geogrid.width,
                        calibration_geogrid.length,
                        calibration_geogrid.epsg)
```
Renames need to be propagated.

Reply:
Marking this as resolved as it was addressed in the commit [here](https://github.com/opera-adt/COMPASS/pull/114/commits/0bf26a93329e76aa8c571c8c2e23dac8e5ceb7ea)